helix-view/src/document.rs

```diff
@@ -134,13 +140,15 @@ impl Document {
     // TODO: passing scopes here is awkward
     // TODO: async fn?
     pub fn load(path: PathBuf, scopes: &[String]) -> Result<Self, Error> {
-        use std::{env, fs::File, io::BufReader};
-        let _current_dir = env::current_dir()?;
+        use std::{fs::File, io::BufReader};
 
-        let file = File::open(path.clone()).context(format!("unable to open {:?}", path))?;
-        let doc = Rope::from_reader(BufReader::new(file))?;
-
-        // TODO: create if not found
+        let doc = if !path.exists() {
+            Ok(Rope::from("\n"))
```

Comment:
Why `\n` but not new rope?

Reply:
The document sort of expects at least one trailing new line or things break -- that's why pressing `d` in a new buffer makes the line number vanish.